projects/Mallard/src/components/article/article-header/opinion-header.tsx

```diff
@@ -151,4 +151,82 @@ const OpinionHeader = ({
     )
 }
 
-export { OpinionHeader }
+const AnalysisHeader = ({
+    byline,
+    bylineImages,
+    headline,
+    image,
+    standfirst,
+}: ArticleHeaderProps) => {
+    const [articleColor] = useArticle()
+    const font = getFont('headline', 1.5, 'light')
+
+    return (
+        <MultilineWrap
+            needsTopPadding
+            style={styles.innerWrap}
+            backgroundColor={color.palette.neutral[97]}
+            byline={
+                <ArticleFader>
+                    <HeadlineTypeWrap>
+                        <View style={[styles.innerWrap]}>
+                            <ArticleStandfirst {...{ standfirst }} />
+                        </View>
+                    </HeadlineTypeWrap>
+                </ArticleFader>
+            }
+        >
+            {image ? (
+                <ArticleFader>
+                    <ArticleImage
+                        style={{
+                            aspectRatio: 1.5,
+                            marginBottom: metrics.vertical / 4,
+                        }}
+                        image={image}
+                    />
+                </ArticleFader>
+            ) : null}
+            <ArticleFader>
+                <View style={styles.flexRow}>
+                    <View style={styles.headlineContainer}>
+                        <HeadlineTypeWrap>
+                            <ArticleHeadline
+                                weight="light"
+                                textStyle={[
+                                    { marginBottom: metrics.vertical },
+                                    font,
+                                ]}
+                            >
+                                <Text
+                                    style={{
+                                        textDecorationLine: 'underline',
+                                        textDecorationColor: articleColor.main,
+                                    }}
+                                >
+                                    {headline}
+                                </Text>
+                                <Text
+                                    style={[
+                                        { color: articleColor.main },
```

Comment:
Inline Styles

Reply:
Can't remove this as it relies on `articleColor` in the component.